doc/faq.md

```diff
@@ -1,3 +1,246 @@
+      return result
+  ```
+
+This code will output
+
+  ```
+  Building adapter [Finished]
+  Waiting for adapter to start [Finished]
+  Running Collect [Finished]
+  Collection Failed: Unexpected collection error: Exception('oops')
+
+  Avg CPU %                     | Avg Memory Usage %         | Memory Limit | Network I/O         | Block I/O
+  ------------------------------+----------------------------+--------------+---------------------+--------------
+  21.1 % (0.0% / 21.1% / 42.2%) | 4.0 % (4.0% / 4.0% / 4.0%) | 1.0 GiB      | 3.24 KiB / 6.67 KiB | 0.0 B / 0.0 B
+
+  Collection completed in 0.45 seconds.
+
+  No collection result was found.
+  No collection result was found.
+  All validation logs written to '/Users/user/management-pack/test-management-pack/logs/validation.log'
+  ```
+As seen above, the Exception is mentioned as the reason for the collection error, and the `No collection result was found` message is also shown.
+Using the collection error message along with the `adapter.log` can help trace the cause of the issue.
+
+### mp-build returns 'Unable to build pak file'
+
+In most cases, this error indicates issues with building the container image. The most probable causes are:
+
+1. Unknown Instruction :
+
+  ```
+  mp-build
+  Building adapter [Finished]
+  Unable to build pak file
+  ERROR: Unable to build Docker file at /Users/user/code/aria_ops/management-packs/test:
+   {'message': 'dockerfile parse error line 7: unknown instruction: COP'}
+
+  ```
+2. A command  inside the Dockerfile failed:
+
+  ```
+  mp-build
+  Building adapter [Finished]
+  Unable to build pak file
+  ERROR: Unable to build Docker file at /Users/user/code/management-packs/test:
+   The command '/bin/sh -c pip3 install -r adapter_requirements.txt --upgrade' returned a non-zero code: 1
+  ```
+The solution for case 1 to fix the typo/command by editing the Dockerfile. For case 2, however, the solution might not be evident at first sight. Since the error
+comes from building the image itself, we can run `docker build .` in the project's root directory and look at the stack trace for clues.
+
+
+### VMware Aria Operations returns 'Unknown adapter type' when setting up new adapter instance
+
+![Example of an 'Unknown Adapter Type' error message for an adapter with type/key 'Testserver'](unknown_adapter_type.png)
+> Example of an 'Unknown Adapter Type' error message for an adapter with type/key 'Testserver'.
+If the pak file installs successfully but errors when creating an account (adapter instance), check that:
+- The Collector/Group the MP is running on is a Cloud Proxy, and
+- Check that the Cloud Proxy supports containerized adapters. Containerized adapter
+  support is supported in VMware Aria Operations version 8.10.0 and later.
+
+### How can I implement Policy and Capacity models? 
+
+Currently, the containerized adapter doest not  have the ability to implement policy settings or capacity models.
```

Comment:
This isn't correct. Policies can still be added, we just can't use them to control what parts of a collection we perform. (for example, there's no `shouldCollect` method for metrics). However, policies can still be added that disable collecting certain metrics - those metrics will just be discarded by Aria Ops rather than not being collected by the adapter in the first place.

Capacities are supported, but not yet by the python-based definitions.